test/fake-intake/Dockerfile

```diff
@@ -0,0 +1,29 @@
+## Based on https://docs.docker.com/language/golang/build-images/
+# syntax=docker/dockerfile:1
+
+## Build
+FROM golang:1.19-buster AS build
+
+WORKDIR /app
+
+COPY go.mod ./
+COPY go.sum ./
+RUN go mod download
+
+COPY fakeintake/*.go ./fakeintake/
+COPY *.go ./
+
+RUN go build -o /build
+
+## Deploy
+FROM gcr.io/distroless/base-debian10
```

Comment:
It's usually not super practical to not have tools like `bash` and `curl`. To keep consistent with other public images, I'd base on `ubuntu:22.04`